lib/buttons-and-links/KRouterLink.vue

```diff
@@ -54,6 +59,13 @@
         type: String,
         required: false,
       },
+      /**
+       * If provided, calls window.history.replaceState() so navigation will not leave history record
+       */
+      replace: {
+        type: Boolean,
+        required: false,
+      },
```

Comment:
To avoid ambiguity and `undefined` input values, component props should either be `required: true` or provide a `default` value.

In this case, the prop should be:

```js
      replace: {
        type: Boolean,
        default: false,
      }
```

Reply:
Thank you! I fixed it in a separate branch as a PR #162. Here. Not sure why that wasn't caught?

Reply:
Ah I missed that - good eye @indirectlylit !